### PR TITLE
Add ability to get IP address of a running container

### PR DESCRIPTION
**Pull request by mheon**
Add an API function to get a container's IP address

### CODE REVIEW

File: libpod/container.go [hunk 1]
```diff
@@ -297,6 +298,23 @@ func (c *Container) LogPath() string {
 	return c.logPath()
 }
 
+// IPAddress returns the IP addres of the container
```
Comment (TomSweeneyRedHat) [inline libpod/container.go]: "addres of the container" to "address of the container".  Surprisingly it looks like lint is not happy with this.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Comment by rhatdan** [comment]
LGTM, We need to get this into podman inspect ASAP.

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
Tests seem green now. Should be good for merge.

**Pull request closed without merging**